src/lib/components/forms/checkbox-group/checkbox-group.component.ts

```diff
@@ -50,73 +49,73 @@ export class CheckboxGroupComponent implements ControlValueAccessor
     @Input()
     public collapsed:boolean = false;
 
-    protected values:Array<any>;
+    public _values:Array<any>;
```

Comment:
Wird nicht im Template verwendet
```suggestion
    private _values:Array<any>;
```